### ISSUE TITLE
Why does the training loss not converge

### ISSUE THREAD

**Issue by rmxhhh** [rbgirshick/py-faster-rcnn#888]
Hi, i want to repetition the vgg16 and zf model (end2end) with pascal voc2007 ,but the loss is not converage and map is only 0.53(zf) and 0.63(vgg16). Then i reduce the lr_mult  in train.prototxt to 0.5 and 1.0,but the result is basically no improvement.  My PC is 1070ti (ubuntu),What should i do?Thank you.

**Comment by ghost** [comment]
Hi,I want to know that what the final loss you got?

**Comment by rmxhhh** [comment]
@SHOW-xiuchao  About six percentage points lower than the author's paper.

**Comment by ghost** [comment]
@rmxhhh What's the loss in paper😂, I didn't find author mention it

**Comment by rmxhhh** [comment]
In the tabel1 and tabel2

**Comment by rmxhhh** [comment]
@SHOW-xiuchao  In the tabel1 and tabel2

**Comment by ghost** [comment]
@rmxhhh I met the problem that the training loss fluctuated all the time. My hyper-parameter is same as the paper.Do you know why?
```
step: 9230 / 40000, total loss: 0.4435, rpn_loss_cls: 0.0446, rpn_loss_box: 0.0276, loss_cls: 0.3173, loss_box: 0.0541
step: 9240 / 40000, total loss: 0.4945, rpn_loss_cls: 0.2028, rpn_loss_box: 0.0131, loss_cls: 0.2622, loss_box: 0.0163
step: 9250 / 40000, total loss: 0.5538, rpn_loss_cls: 0.1154, rpn_loss_box: 0.0392, loss_cls: 0.3699, loss_box: 0.0294
step: 9260 / 40000, total loss: 1.2168, rpn_loss_cls: 0.3837, rpn_loss_box: 0.1008, loss_cls: 0.7106, loss_box: 0.0216
step: 9270 / 40000, total loss: 0.8166, rpn_loss_cls: 0.1556, rpn_loss_box: 0.0254, loss_cls: 0.5997, loss_box: 0.0359
step: 9280 / 40000, total loss: 0.6296, rpn_loss_cls: 0.0559, rpn_loss_box: 0.0227, loss_cls: 0.5061, loss_box: 0.0448
step: 9290 / 40000, total loss: 0.4234, rpn_loss_cls: 0.0476, rpn_loss_box: 0.0149, loss_cls: 0.3324, loss_box: 0.0285
step: 9300 / 40000, total loss: 0.3778, rpn_loss_cls: 0.1024, rpn_loss_box: 0.0132, loss_cls: 0.2322, loss_box: 0.0300
step: 9310 / 40000, total loss: 0.9004, rpn_loss_cls: 0.2039, rpn_loss_box: 0.0695, loss_cls: 0.5671, loss_box: 0.0600
step: 9320 / 40000, total loss: 0.4982, rpn_loss_cls: 0.1066, rpn_loss_box: 0.0069, loss_cls: 0.3376, loss_box: 0.0470
step: 9330 / 40000, total loss: 0.6691, rpn_loss_cls: 0.2639, rpn_loss_box: 0.0238, loss_cls: 0.3330, loss_box: 0.0484
step: 9340 / 40000, total loss: 1.0502, rpn_loss_cls: 0.3845, rpn_loss_box: 0.0803, loss_cls: 0.5368, loss_box: 0.0486
step: 9350 / 40000, total loss: 0.5347, rpn_loss_cls: 0.0648, rpn_loss_box: 0.0331, loss_cls: 0.4063, loss_box: 0.0305
step: 9360 / 40000, total loss: 0.6605, rpn_loss_cls: 0.1370, rpn_loss_box: 0.0860, loss_cls: 0.4082, loss_box: 0.0293
step: 9370 / 40000, total loss: 0.5732, rpn_loss_cls: 0.0668, rpn_loss_box: 0.0143, loss_cls: 0.4494, loss_box: 0.0427
step: 9380 / 40000, total loss: 0.8108, rpn_loss_cls: 0.2477, rpn_loss_box: 0.0267, loss_cls: 0.4610, loss_box: 0.0753
step: 9390 / 40000, total loss: 0.6381, rpn_loss_cls: 0.1227, rpn_loss_box: 0.0400, loss_cls: 0.4388, loss_box: 0.0366
step: 9400 / 40000, total loss: 0.3847, rpn_loss_cls: 0.0598, rpn_loss_box: 0.0060, loss_cls: 0.3012, loss_box: 0.0177
step: 9410 / 40000, total loss: 0.6791, rpn_loss_cls: 0.1005, rpn_loss_box: 0.0024, loss_cls: 0.5525, loss_box: 0.0236
step: 9420 / 40000, total loss: 0.5571, rpn_loss_cls: 0.3315, rpn_loss_box: 0.0190, loss_cls: 0.1989, loss_box: 0.0077
step: 9430 / 40000, total loss: 0.7480, rpn_loss_cls: 0.0764, rpn_loss_box: 0.0102, loss_cls: 0.6242, loss_box: 0.0373
step: 9440 / 40000, total loss: 0.4599, rpn_loss_cls: 0.0802, rpn_loss_box: 0.0214, loss_cls: 0.3053, loss_box: 0.0531
step: 9450 / 40000, total loss: 0.4007, rpn_loss_cls: 0.0637, rpn_loss_box: 0.0086, loss_cls: 0.2949, loss_box: 0.0335
step: 9460 / 40000, total loss: 0.8419, rpn_loss_cls: 0.2803, rpn_loss_box: 0.1721, loss_cls: 0.3669, loss_box: 0.0225
step: 9470 / 40000, total loss: 0.5050, rpn_loss_cls: 0.1251, rpn_loss_box: 0.0126, loss_cls: 0.3137, loss_box: 0.0536
step: 9480 / 40000, total loss: 0.4077, rpn_loss_cls: 0.0480, rpn_loss_box: 0.0169, loss_cls: 0.3082, loss_box: 0.0346
step: 9490 / 40000, total loss: 0.6010, rpn_loss_cls: 0.2832, rpn_loss_box: 0.1072, loss_cls: 0.1958, loss_box: 0.0148
step: 9500 / 40000, total loss: 0.4630, rpn_loss_cls: 0.0336, rpn_loss_box: 0.0208, loss_cls: 0.3918, loss_box: 0.0168
step: 9510 / 40000, total loss: 0.5703, rpn_loss_cls: 0.0651, rpn_loss_box: 0.0501, loss_cls: 0.4127, loss_box: 0.0424
step: 9520 / 40000, total loss: 0.2905, rpn_loss_cls: 0.0201, rpn_loss_box: 0.0178, loss_cls: 0.2071, loss_box: 0.0456
step: 9530 / 40000, total loss: 0.5912, rpn_loss_cls: 0.1524, rpn_loss_box: 0.0420, loss_cls: 0.3695, loss_box: 0.0272
```
